testing/proxy/proxy_test.go

```diff
@@ -0,0 +1,215 @@
+//go:build rely_docker
+
+// package proxy is an example of how the testing package can be used. It tests
+// three contracts that are used to form a proxy pattern. An explanation of
+// proxy contracts in Solidity can be found here:
+// https://www.cyfrin.io/blog/upgradeable-proxy-smart-contract-pattern
+package proxy
+
+import (
+	"context"
```

Comment:
This is nbd.  
I notice that the tab width in you IDE is configured to 8 spaces, that's why your newly created files like this one `       "context"`.

Can you try add `tab_width = 4` to `[{*.go,*.go2}]` sestion in `.editorconfig` file, see if it chages?

Reply:
Just to make this clear. The purpose of `.editorconfig` is to make sure all developers have same view of the source code, to avoid unnecessary pr conflicts.